compiler_opt/rl/generate_bc_trajectories.py

```diff
@@ -154,13 +172,15 @@ def __init__(
       explore_on_features: Optional[Dict[str, Callable[[tf.Tensor],
                                                        bool]]] = None,
   ):
-    self.replay_prefix = replay_prefix
-    self.policy = policy
-    self.explore_policy = explore_policy
-    self.curr_step = 0
-    self.explore_step = 0
-    self.gap = np.inf
-    self.explore_on_features = explore_on_features
+    self.explore_step: int = len(replay_prefix) - 1
```

Comment:
`self._explore_step`

Reply:
ah, OK. I see what is also happening. Can you separate the stylistic changes in their own PR, and just submit it once it passes the CI (no review needed, I suggest naming it like "[NFC] add `_` for member fields"). Then merge this.

This helps both with this review and later: the mechanical change doesn't interfere (and one knows not to spend much time looking at it in the future, when we want to understand something)